src/components/Map/StaticMap.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { MouseEvent, Component } from "react";
+import { Image } from "../Image/Image";
+import { Button } from "../Button/Button";
+import { MAP_KEY } from "./constant";
+import "./StaticMap.scss";
+
+/**
+ * MarkerColor
+ * Color (24-bit) for markers on the map
+ */
+export enum MarkerColor {
+  Blue = "58a5fc",
+  Grey = "82aabf",
+}
+
+/**
+ * Marker
+ * lat: latitude of the marker
+ * long: longitude of the marker
+ * color: (Optional) Color of the marker
+ */
+export interface Marker {
+  lat: number;
+  long: number;
+  color?: MarkerColor;
+}
+
+/**
+ * StaticMapProps
+ * mapWidth: width of the map (maximum is 640px)
+ * mapHeight: height of the map  (maximum is 640px)
+ * mapScale: scale of the map (1 or 2), 2 will double the resolution
```

Comment:
Just a question: Will the map be autoscaled on default to fit all the markers in markers array?

Reply:
Yuppp, the API automatically determine the center and zoom level to include all the markers. The scale is used to determine the resolution of the map, it doesn't affect the zoom level.